### PR TITLE
Implement get all vendors endpoint

### PR DESCRIPTION
**Pull request by iphyokafor**
#### What does this PR do?
This feature retrieves all registered vendors on the Dae-Alright application.
#### How should this manually be Tested?
- `git clone` __https://github.com/devcareer/Dae-Alright-Backend__
- `cd` **Dae-Alright-Backend**
- `git checkout <branch-name>`
- `npm install`
- `npm test`

#### Screenshots 
Screenshot (39)

### CODE REVIEW

**Comment by meetKazuki** [comment]
[Retrieve all Food Vendors](https://trello.com/c/vsLUYpaW/11-retrieve-all-food-vendors)

File: src/routes/vendor.route.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// import getVendors from '../controllers/vendors';
+
+
+// const router = Router();
+
+// router.get('/vendors', getVendors);
+
+
+// export default router;
+
```
Comment (meetKazuki) [inline src/routes/vendor.route.js]: Remove the comments.

**Review by meetKazuki — COMMENTED** [review]
Are we using `bcrypt` or `bcryptjs`?

File: src/database/seeders/20191028063855-demo-vendor.js [hunk 2]
```diff
@@ -1,113 +1,110 @@
...
-        email: 'ciggalden9@paypal.com',
-        password: await bcrypt.hash('12345', await bcrypt.genSalt(+process.env.SALT)),
-        location: '2 Boyd Alley',
-        phone: '874-612-9083',
-        created_at: new Date(),
-        updated_at: new Date(),
-      }
-    ],
-    {}
+    'Vendors', [{
```
Comment (TundeMercy) [inline src/database/seeders/20191028063855-demo-vendor.js:7]: Why changing to this format?

File: src/database/seeders/20191028063855-demo-vendor.js [hunk 3]
```diff
@@ -1,113 +1,110 @@
 import uuidv4 from 'uuid/v4';
-import bcrypt from 'bcrypt';
+import bcrypt from 'bcryptjs';
```
Comment (TundeMercy) [inline src/database/seeders/20191028063855-demo-vendor.js]: Why `bcryptjs` instead of `bcrypt` that we have before?

Comment (supercede) [reply src/database/seeders/20191028063855-demo-vendor.js]: Sandra, you probably have issues running bcrypt on your laptop, I did too. What you'll have to do is:
1. Install windows-build-tools and python2.7
2. Install Visual Studio with C++ worktools
That's how I got around the issue

**Comment by meetKazuki** [comment]
Rebase please

**Pull request merged**